packages/react-native/ReactCommon/react/nativemodule/core/platform/ios/ReactCommon/RCTTurboModuleManager.mm

```diff
@@ -187,6 +188,7 @@ static Class getFallbackClassFromName(const char *name)
 
 @implementation RCTTurboModuleManager {
   std::shared_ptr<CallInvoker> _jsInvoker;
+  facebook::jsi::Runtime *_runtime;
```

Comment:
good catching @philIip! i've passed runtime to `provideTurboModule:` without storing the runtime in the class. also moving the installation earlier and right after `getTurboModule:`. that's a point to ensure running on js thread, even requiresMainQueueSetup is true.

aside from that, i think on ios and android implementation now still passing runtime pointer ([android](https://github.com/Kudo/react-native/blob/a7b7bf632af664e3bdd844605c54741a0e5d1d5c/packages/react-native/ReactAndroid/src/main/jni/react/turbomodule/ReactCommon/TurboModuleManager.cpp#L344) / [ios](https://github.com/Kudo/react-native/blob/a7b7bf632af664e3bdd844605c54741a0e5d1d5c/packages/react-native/ReactCommon/react/nativemodule/core/platform/ios/ReactCommon/RCTTurboModuleManager.mm#L935)). do you think whether this would be a concern? at least at `provideTurboModule:` and `createTurboModuleProvider()`, this is when people call `TurboModuleRegistry.get()` from js, so runtime should be valid at this point.
we could maybe turn RuntimeExecutor into BindingsInstaller without passing runtime, but we would at least need an UnbufferedRuntimeExecutor. Otherwise it may be too late to install the js bindings.